MAINTAINERS.yml

```
@@ -4716,6 +4716,17 @@ Testing with Renode:
   labels:
     - "area: Renode"
 
+Test Configuration Files:
+  status: odd fixes
+  collaborators:
+    - nashif
+    - JarmouniA
+  files:
+    - samples/*/sample.yaml
+    - tests/*/testcase.yaml
```

You'll need to use the `files-regex`

```suggestion
  files-regex:
    - ^samples/.*/sample.yaml$
    - ^tests/.*/testcase.yaml$
```